target/linux/qualcommbe/ipq95xx/base-files/etc/hotplug.d/firmware/12-ath12k-caldata

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+[ -e /lib/firmware/$FIRMWARE ] && exit 0
+
+. /lib/functions/caldata.sh
+
+board=$(board_name)
+
+case "$FIRMWARE" in
+"ath12k/QCN9274/hw2.0/cal-pci-0001:01:00.0.bin")
+	case "$board" in
+	tplink,be800v1)
+		caldata_extract "0:ART" 0x26800 0x2D000
```

Comment:
Can we use NVMEM for this or does ath12k not support this? You can look at some devices that use ath11k and NVMEM in device tree for inspiration.

Reply:
Hi @Rondom I didn't find a way to load caldata from NVMEM, currently it can be loaded from file only.